internal/apps/apps.go

```diff
@@ -42,6 +42,11 @@ func (c *Config) RunHTTP(handler func() http.Handler) {
 		os.Setenv("DD_PROFILING_EXECUTION_TRACE_PERIOD", "1s")
 	}
 
+	// Enabled runtime metrics v2 by default
+	if v := os.Getenv("DD_RUNTIME_METRICS_V2_ENABLED"); v == "" {
+		os.Setenv("DD_RUNTIME_METRICS_V2_ENABLED", "true")
+	}
```

Comment:
Why do this rather than change line 387 in ddtrace/tracer/option.go to `c.runtimeMetricsV2 = internal.BoolVal("DD_RUNTIME_METRICS_V2_ENABLED", true)`?

Reply:
This feature is not meant to be enabled for customers yet. We want to add it to dd-trace-go behind an env var so that we can experiment with it in the backend go repos at Datadog. If that goes well, we'll document and advertise this feature and probably turn it on by default.